pages/en/blog/announcements/v22-release-announce.md

```diff
@@ -0,0 +1,96 @@
+---
+date: '2024-04-17T15:45:00.000Z'
+category: announcements
+title: 'Node.js 22 is now available!'
+layout: blog-post
+author: The Node.js Project
+---
+
+We're excited to announce the release of Node.js 22!
+Highlights include require()ing ESM graphs, WebSocket client, updates of the V8 JavaScript engine, and more!
+
+As a reminder, Node.js 22 will enter long-term support (LTS) in October, but until then, it will be the "Current" release for the next six months.
```

Comment:
```suggestion
Node.js 22 will enter long-term support (LTS) in October, but until then, it will be the "Current" release for the next six months.
```

Reply:
I think it's fine to keep it, since its actually a reminder

Reply:
For most people reading it, it probably won't be a reminder.